### ISSUE TITLE
Cursor control continued

### ISSUE THREAD

**Issue by duencil** [GreatArcStudios/unitypausemenu#6]
This is continuing unaddressed issues from https://github.com/GreatArcStudios/unitypausemenu/issues/2

Cancel functionality was added, but that was just one of three issues with cursor control. We still have outstanding:

1 When a menu appears, nothing is highlighted by default. 
2 When a slider or toggle option is selected, nothing is highlighted.You can see this in the Graphics menu trying to navigate over the options using cursors / controller buttons

**Comment by GreatArcStudios** [comment]
This was something I had a problem with myself. What I did the first time was to set the default selected to a particular thing each time a the panels changed, which worked for keyboard/joystick control, but it didn't highlight anything. Sorry that I didn't fix it with the release, but I felt that 1.0 was long over due so I decided to release it with out your suggestion. I'll try to fix it now.

**Comment by GreatArcStudios** [comment]
I'll keep it open until I fix it.

**Comment by duencil** [comment]
Yeah, I think one of the problems I am facing is in the graphics options, the toggles and sliders have color tints to indicate normal / highlighted status, and both colors are the same, so its impossible to see which is highlighted.  

And then the advanced options are on a scrolling panel, which I'm not sure works well with cursor control.  

Since I'm grabbing your source control, but not your prefab changes, I'll probably not be able to benefit from your fixes if you address this.  But I'm interested in any solutions you may have, so I can implement the same fixes here.
